src/components/scroll/znkScrollDrv.js

```diff
@@ -79,6 +82,8 @@
                         currMousePoint = null;
                     }
                     function mouseDownHandler(evt){
+                        //$log.debug('mouse down',evt.pageX);
```

Comment:
Yeah, i use when i'm working on this directive but i don't want it make noise in the act app.